types/office-js-preview/index.d.ts

```diff
@@ -957,7 +957,7 @@ declare namespace Office {
          *
          * @remarks
          *
-         * **{@link https://learn.microsoft.com/office/dev/add-ins/outlook/understanding-outlook-add-in-permissions | Minimum permission level}**: `Restricted`
+         * **{@link https://learn.microsoft.com/office/dev/add-ins/outlook/understanding-outlook-add-in-permissions | Minimum permission level}**: **restricted**
```

Comment:
Minor nitpicky question, but why is this lowercase instead of "Restricted"? I'd think the capital would make it easy to skim.